solidity/test/bridge/Bridge.Frauds.test.ts

```diff
@@ -0,0 +1,1528 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+import { ethers, helpers, waffle } from "hardhat"
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import chai, { expect } from "chai"
+import { Transaction } from "ethers"
```

Comment:
I think we usually use `ContractTransaction`. It has some additional fields that can be useful for testing.

Reply:
Done in 8e8feed7b7b55753b47ade47d2ac82db25e77519.